package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "git-task",
+  "version": "1.0.0",
+  "description": "working with git",
+  "main": "main.js",
+  "scripts": {
+    "build": "tsc --project ./tsconfig.json",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "ts-node ./src/main.ts"
+  },
+  "keywords": [
+    "git"
+  ],
+  "author": "Vadim Stadnichenko",
+  "license": "ISC",
+  "dependencies": {
+    "bcrypt": "^5.0.1",
+    "body-parser": "^1.19.2",
+    "dotenv": "^16.0.0",
+    "express": "^4.17.3",
+    "jsonwebtoken": "^8.5.1",
+    "mssql": "^8.1.0",
+    "mysql": "^2.18.1",
+    "mysql2": "^2.3.3",
```

Comment:
do we use all of these libs?

Reply:
No, we don't. I'm forgot about to clear this libs